### ISSUE TITLE
Unable to use multiline secrets

### ISSUE THREAD

**Issue by Siegrift** [api3dao/airnode#1244]
https://api3workspace.slack.com/archives/C02AYRX8D89/p1655307769484259

**Comment by kolenic-martin** [comment]
Hi @Siegrift, what kind of a private key can I use for this? Is gcp.json okay for that, provided that I insert some line breaks into the key?

**Comment by kolenic-martin** [comment]
Hi, @Siegrift, so I've tested this issue as we've discussed in the call, but for reference purposes, let me just outline it here:

- I created a mock API with Beeceptor that would accept a multiline string from the secrets file
- I then introduced a new secret in secrets.env, called SECRET_MESSAGE, which I had contain some Lorem ipsum with a couple linebreaks, wrapped in quotes.
- In config.json, I passed this as a request parameter.
- The deployment went just fine, no complaints from the validator.
- Beeceptor showed the message that was passed as the parameter just fine as well.

This is thus good to go, I'm moving the issue :)
